content/1-reactivity/2-update-state/vue2/Name.vue

```diff
@@ -0,0 +1,16 @@
+<script>
+export default {
+	data() {
+		return {
+			name: 'John',
+		};
+	},
+	create() {
```

Comment:
`created` instead of `create` https://v2.vuejs.org/v2/guide/instance.html#Instance-Lifecycle-Hooks